controllers/admin/AdminModulesController.php

```diff
@@ -1447,23 +1447,23 @@ public function initModal()
         $this->modals[] = array(
             'modal_id' => 'moduleTradLangSelect',
             'modal_class' => 'modal-sm',
-            'modal_title' => $this->l('Translate this module'),
+            'modal_title' => $this->trans('Translate this module'),
             'modal_content' => $modal_content,
         );
 
         $modal_content = $this->context->smarty->fetch('controllers/modules/' . (($this->context->mode == Context::MODE_HOST) ? 'modal_not_trusted_blocked.tpl' : 'modal_not_trusted.tpl'));
         $this->modals[] = array(
             'modal_id' => 'moduleNotTrusted',
             'modal_class' => 'modal-lg',
-            'modal_title' => ($this->context->mode == Context::MODE_HOST) ? $this->l('This module cannot be installed') : $this->l('Important Notice'),
+            'modal_title' => ($this->context->mode == Context::MODE_HOST) ? $this->trans('This module cannot be installed') : $this->trans('Important Notice'),
             'modal_content' => $modal_content,
         );
 
         $modal_content = $this->context->smarty->fetch('controllers/modules/modal_not_trusted_country.tpl');
         $this->modals[] = array(
             'modal_id' => 'moduleNotTrustedCountry',
             'modal_class' => 'modal-lg',
-            'modal_title' => $this->l('This module is Untrusted for your country'),
+            'modal_title' => $this->trans('This module is Untrusted for your country'),
```

Comment:
Weird wording.

Reply:
Do I keep it as it is?

Reply:
I would rather write it _Untrusted module for your country_ but it does not change much. Anyway, this string already exists so we can keep it as-is if it is the only change required.